### DISCUSSION TITLE
Documentation for V1.0

### DISCUSSION THREAD

**Discussion by g-walley** [g-walley/cegpy#93] [General]
@ashenvi10 could you help write a paragraph for the introduction to the package?
Maybe base it on this.

**Cegpy** (/segpaɪ/) is a Python library for creating Chain Event Graphs.
It is based on the network modelling package networkx.

**Comment by ashenvi10** [comment]
Cegpy (/segpaɪ/) is a Python package for working with Chain Event Graphs. It supports learning the graphical structure of a Chain Event Graph from data, encoding of parametric and structural priors, estimating its parameters, and performing inference. 

It is built on top of the Python network modelling package NetworkX.
